src/components/input/input.stories.ts

```diff
@@ -0,0 +1,258 @@
+        disable: true,
+      },
+    },
+    validationText: {
+      control: { type: 'text' },
+      description: 'Property for describing validation',
+    },
+    disabled: {
+      control: { type: 'boolean' },
+      options: [true, false],
+      description: 'Property for disabled boolean',
+    },
+
+    click: {
+      table: {
+        disable: true,
+      },
+    },
+
+    change: {
+      table: {
+        disable: true,
+      },
+    },
+
+    'update:modelValue': {
+      table: {
+        disable: true,
+      },
+    },
+  },
+} as Meta<typeof FoamInput>;
+
+const Template: StoryFn<typeof FoamInput> = (args, { argTypes }) => {
+  return {
+    components: { FoamInput },
+    setup() {
+      return { args, argTypes };
+    },
+    template: '<foam-input v-bind="args" placeholder="placeholder" />',
+  };
+};
+
+export const Default = Template.bind({});
+
+
+export const Variants: StoryFn<typeof FoamInput> = (args, { argTypes }) => ({
+  components: { FoamInput },
+  setup() {
+    return { args, argTypes };
+  },
+  template: `
+  <div style="display: flex; flex-direction:column; flex-wrap: wrap;">
+    <foam-input v-bind="args" v-for="variant in argTypes.variant.options" :key="variant" :label="variant" :variant="variant" placeholder ="placeholder" />
+  </div>
+`,
+});
+
+Variants.argTypes = {
+  variant: {
+    table: {
+      disable: true,
+    },
+  },
+  activeDropdown: {
+    table: {
+      disable: true,
+    },
+  },
+  dropdown: {
+    table: {
+      disable: true,
+    },
+  },
+  placeholder: {
+    table: {
+      disable: true,
+    },
+  },
+  text: {
+    description: 'Property for text content inside the input',
+  },
+};
+
+export const Sizes: StoryFn<typeof FoamInput> = (args, { argTypes }) => ({
+  components: { FoamInput },
+  setup() {
+    return { args, argTypes };
+  },
+  template: `
+  <div style="display: flex; flex-direction:column; flex-wrap: wrap;">
+    <foam-input v-bind="args" v-for="size in argTypes.size.options" :key="size" :label="size" :size="size" placeholder ="placeholder" />
+  </div>
+`,
+});
+
+Sizes.argTypes = {
+  variant: {
+    table: {
+      disable: true,
+    },
+  },
+  activeDropdown: {
+    table: {
+      disable: true,
+    },
+  },
+  dropdown: {
+    table: {
+      disable: true,
+    },
+  },
+  placeholder: {
+    table: {
+      disable: true,
+    },
+  },
+};
+
+export const Validation: StoryFn<typeof FoamInput> = (args, { argTypes }) => {
+  return {
+    components: { FoamInput },
+    setup() {
+      return { args, argTypes };
+    },
+    template: `
+    <div style="display: flex; flex-direction: column;">
+    <foam-input  variant="text" helpertext="helpertext" validationText="validationText" :isValid=false v-bind="args"  />
+    <foam-input  variant="text" 
+    helpertext="helpertext"  validationText="validationText"  :isValid=true v-bind="args"  />
+    </div>
+  `,
+  };
+};
+
+export const Accordian: StoryFn<typeof FoamInput> = () => {
```

Comment:
It should be spelled "Accordion" instead of "Accordian" 🙃 so change this everywhere

Reply:
There are a couple left, in input.md and input.stories.ts